sandbox/__test__/user_details.spec.js

```diff
@@ -0,0 +1,215 @@
+import request from "supertest"
+import * as  fs from 'fs'
+import { setup } from './helpers.js'
+
+describe("/comms/channels/app/supplier/nhsapp/accounts", () => {
+    let env;
+    let server;
+
+    before(function () {
+        env = process.env;
+        server = setup();
+    });
+
+    after(function () {
+        process.env = env;
+        server.close();
+    });
+
+    it("returns a service ban (403) when the user is banned", (done) => {
+        request(server)
+            .get("/comms/channels/app/supplier/nhsapp/accounts")
```

Comment:
This is the path on the APIM side, on sandbox I believe it should look more like `/api/channels/app/supplier/nhsapp/accounts` to reflect what we will call on API Gateway.

Reply:
Updated